OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -433,7 +436,14 @@ public void Tick(PlayerResources pr)
 			{
 				var time = Queue.GetBuildTime(Item);
 				if (time > 0)
+				{
 					RemainingTime = TotalTime = time;
+					framesToSkip = TotalTime - TotalCost;
```

Comment:
Can you add a comment similar to the commit description explaining what's going on here?  At first glance it looks like a bug to be relating TotalTime with TotalCost (and tbh i'm still not convinced whether this will work for all custom build time / queue multiplier values).